src/Util/Crypto.php

```diff
@@ -283,13 +283,13 @@ private static function decryptAES256CBC($data, $key, $iv)
 
 	/**
 	 * Encrypt a string with 'aes-256-ctr' cipher method.
-	 * 
+	 *
 	 * Ported from Hubzilla: https://framagit.org/hubzilla/core/blob/master/include/crypto.php
-	 * 
+	 *
 	 * @param string $data
-	 * @param string $key   The key used for encryption.
-	 * @param string $iv    A non-NULL Initialization Vector.
```

Comment:
Those white spaces are actually wanted, so all the parameter comments are aligned.